x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/generation_modal.tsx

```diff
@@ -99,7 +99,11 @@ export const useGeneration = ({
           return;
         }
         const categorizationRequest: CategorizationRequestBody = {
-          ...ecsRequest,
+          packageName: integrationSettings.name ?? '',
```

Comment:
Yeah... The related request will have same input as categorizationRequest. This changed from ECS Request because ECS request has encodedstring and categorization request gets decoded and parsed JSON samples

Reply:
Changed this back , since we decided to go with new API call for `analyseLogs`